### ISSUE TITLE
Cancelling 'download from url' clears picture

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Risehat/freemedforms#239]
```
What steps will reproduce the problem?
1. Click on the patients photo in the Identity section, select Download from 
URL option
2. Cancel downloading from URL

What is the expected output? What do you see instead?
The previous picture should stay, however it gets cleared instead.

What version of the product are you using? On what operating system?
0.8.0-precise1 on Ubuntu 12.04
```

Original issue reported on code.google.com by `vpereto...@gmail.com` on 11 Dec 2012 at 5:26

**Comment by GoogleCodeExporter** [comment]
Original comment by `christian.a.reiter@gmail.com` on 11 Dec 2012 at 7:34
- Changed state: **Started**
- Added labels: **Priority-Critical**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
yes, this is exactly what I think :D 
But I fully agree that % is not enough for a good ui management, and I still 
think to update this to 1/1000.
```

Original comment by `eric.mae...@gmail.com` on 11 Dec 2012 at 4:58

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision c6b6bad11f0b.
```

Original comment by `christian.a.reiter@gmail.com` on 12 Dec 2012 at 12:43
- Changed state: **Fixed**
